frontend/src/app.js

```diff
@@ -0,0 +1,48 @@
+
+import 'babel-polyfill/browser';
+import es6Promise from 'es6-promise';
+import 'isomorphic-fetch';
+import 'l20n';
+import Raven from 'raven-js';
+
+import './app/lib/ga-snippet';
+import config from './app/config';
+
+import error from './pages/error.js';
+import experiment from './pages/experiment.js';
+import experiments from './pages/experiments.js';
+import home from './pages/home.js';
+import legacy from './pages/legacy.js';
+import onboarding from './pages/onboarding.js';
+import retire from './pages/retire.js';
+import share from './pages/share.js';
+
+es6Promise.polyfill();
+Raven.config(config.ravenPublicDSN).install();
+
+const routes = {
+  error,
+  experiment,
+  experiments,
+  home,
+  legacy,
+  onboarding,
+  retire,
+  share
+};
+
+let path = window.location.pathname;
```

Comment:
Also, this might be a crazy idea / idea for a future PR—what if each static-rendered page had something like:
```html
<body data-page-name="experiment" data-page-param="snooze-tabs">
```
The server naturally does the routing for us, and this block of code could be stripped down to this:
```
const name = document.body.dataset.pageName;
const param = document.body.dataset.pageParam;
if (name in routes) {
  routes[name](param);
} else {
  routes.error();
}
```